backend/controllers/followups.js

```diff
@@ -4,7 +4,10 @@ const Followup = require("mongoose").model("Followup")
 const situation = require("./answers")
 const pollResult = require("../lib/mattermost-bot/poll-result")
 
-const excludeFields = ["surveys.accessToken"].join(" -").replace(/^/, "-")
+// TODO next line is to be updated once tokens are used globally
+const excludeFields = ["accessToken", "surveys.accessToken"]
+  .join(" -")
+  .replace(/^/, "-")
```

Comment:
Le code ci-dessus ne me parait pas utile 
pourquoi ne pas directement faire  : 
```suggestion
const excludeFields = "-accessToken -surveys.accessToken"
```

Reply:
On en avait discuté à l'oral mais pour rappel, mon point de vue est qu'il est facile de faire une typo  / oublier de mettre un `-` devant un champ qu'on veut exclure. Du coup même si ce code est plus complexe il évite des situations où on expose un token.

Reply:
On en avait discuté à l'oral mais pour rappel, mon point de vue est qu'il est facile de faire une typo  / oublier de mettre un `-` devant un champ qu'on veut exclure. Du coup même si ce code est plus complexe il évite des situations où on expose un token.